app/src/main/java/com/github/fribourgsdp/radio/game/GameActivity.kt

```diff
@@ -68,6 +68,7 @@ open class GameActivity : AppCompatActivity(), GameView, Timer.Listener {
     protected lateinit var voiceChannel: VoiceIpEngineDecorator
 
     private lateinit var playerGameHandler: PlayerGameHandler
+    private lateinit var hostGameHandler : HostGameHandler
```

Comment:
Instead of doing a late init here, since not everyone has it, I would make it nullable.
Then in `onDestroy()`, line 112, you can simply do a null check: `hostGameHandler?.unlinkFromDatabase()`. I think this is a better implementation since some user won't have some `lateinit` not initialized.

Reply:
corrected